.gitignore

```diff
@@ -91,3 +91,5 @@ ModulesMaintainers.txt
 Nits.txt
 Pylint.txt
 YAMLLint.txt
+/dts/arm/microchip/pic32cxsg
+pic32cx_sg41_cult
```

Comment:
Hi @pmcgaugh ,

It is not clear the use case here. If you need some files to autogenerate other maybe a regular expression could be better to cover all the cases.

Reply:
@pmcgaugh @nandojve I did not add these lines, what do you want me to do ?
A meeting would be a option.

Reply:
We can schedule a meeting to sync about. I can try after 18h ( UTC+2 ).